core/src/epicli/data/common/ansible/playbooks/roles/kubernetes_master/templates/azure/k8s-persistent-volume.yml.j2

```diff
@@ -1,10 +1,14 @@
 apiVersion: "v1"
 kind: "PersistentVolume"
 metadata:
-  name: "pv0001" 
+metadata:
+  name: {{ specification.storage.name }}
+  labels:
+    name: {{ specification.storage.name }}
 spec:
+  storageClassName: azurefile
   capacity:
-    storage: "15Gi" 
+    storage: {{ specification.storage.capacity }}Gi 
```

Comment:
Are we sure we want the suffix hard-coded to Gibibytes?

Reply:
As per Tomasz request yes.

Reply:
Ok, then